lighttpd/README.md

```diff
@@ -0,0 +1,32 @@
+# Lighttpd - Webserver for remote access to camera feed on BruinBot
+
+# Installation
+
+Download from here: https://www.lighttpd.net/download/  
+Alternatively on macOS, if you have homebrew installed, run `brew install lighttpd` 
+
+# Modifying config file
+
+Replace the config file located at `/usr/local/etc/lighttpd/lighttpd.conf` with the config file in this directory
+
+# Creating index.html
+
+There should be an existing index.html file located at `/usr/local/var/www/index.html`. If it doesn't exist, create an index.html at the specified path. Copy the code from the index.html located in this directory to your local index.html file
+
+# Running Lighttpd
+
+```
+lighttpd -f /usr/local/etc/lighttpd/lighttpd.conf
+```
+
+You should now be able to visit `localhost:8090` to view the webserver.
+
+To restart the server for any reason:
+```
+lighttpd -f /usr/local/etc/lighttpd/lighttpd.conf restart
```

Comment:
@Saagar3690 does this command provided work?
I think it should be `kill -USR1 $(cat /usr/local/var/lighttpd/lighttpd.pid)`